config/crd/bases/numaplane.numaproj.io_pipelinerollouts.yaml

```diff
@@ -458,6 +458,10 @@ spec:
                       name:
                         description: Name of the upgrading child
                         type: string
+                      originalScaleDefinitions:
+                        items:
+                          type: string
+                        type: array
```

Comment:
these are the "full scale definitions", defined as an array, since it's per vertex